editor/src/styles/_sites.scss

```diff
@@ -13,3 +13,12 @@ berta-site {
   background-color: $white;
   display: block;
 }
+
+.active{
+  background-color: $grey-10;
+}
+
+.link:hover span, 
+.link:hover a {
+  color: $berta-accent !important;
```

Comment:
Try to avoid using `!important`. This is not necessary here, rewrite the css property definition.